### DISCUSSION TITLE
Точка входа в процедуру CreateDXGIFactory2

### DISCUSSION THREAD

**Discussion by benspinoza** [ScanTailor-Advanced/scantailor-advanced#57] [Q&A]
**v1.0.18** при запуске пишет: "Точка входа в процедуру CreateDXGIFactory2 не найдена в библиотеке DLL dxgi.dll."
Не запускается. Win7x64

**Comment by darnn** [comment]
Have you tried any of this?
https://www.softwaretesttips.com/procedure-entry-point-createdxgifactory2-could-not-be-located/

**Comment by benspinoza** [comment]
I've tried to put  dxgi.dll of different versions into the program folder, then tried the method with ReShade 5.5.2. No effect.

**Comment by darnn** [comment]
Can you remove it altogether? I don't have it at all...

**Comment by benspinoza** [comment]
Win7 does not allow to delete dxgi.dll. There are 6 versions of it in system folders
C:\Windows\System32\dxgi.dll
C:\Windows\SysWOW64\dxgi.dll
C:\Windows\winsxs\amd64_microsoft-windows-directx-dxgi_31bf3856ad364e35_6.1.7601.17514_none_98a47dc2352c4409\dxgi.dll
etc.

**Comment by ghost** [comment]
@benspinoza Are your GPU drivers installed properly? Try using dxdiag to check. (Click Windows+R and type dxdiag and press enter, go to screen and see what it says).

**Comment by rddim** [comment]
I have the same error on Win 7. Qt6 does not support Win 7.

**Comment by mara004** [comment]
> I have the same error on Win 7. Qt6 does not support Win 7.

In that case, perhaps releasing an alternative build with Qt5 would be an option, if it's not too much work to set up?
If you had release automation, I'd guess it might be relatively simple to adapt, but it looks like there is no build script / GH workflow yet?

**Comment by ghost** [comment]
@mara004 I'm actually working on that. I also believe this is not a bug with ScanTailor, so I'll probably convert it to a discussion.

**Reply by mara004** [reply]
> I also believe this is not a bug with ScanTailor

If it's true the above error is because of the Qt6 <-> Win7 conflict (which I guess it is), then it's still sort of a bug with ScanTailor ("Binary releases don't run on Windows <= 7 due to Qt6 incompatibility")

That said, we might want to document minimum system requirements.

**Reply by ghost** [reply]
@mara004 Well, on the build instructions it is said that Qt6 is not supported by Windows 7. Also, the original author has not been here for a while, so I don't know if this issue is still relevant.
